src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -257,13 +261,75 @@
+        :on-done (fn [{:keys [success? get-parsed-response status-text]}]
+                     (if success?
+                       (let [response (get-parsed-response)]
+                            (endpoints/call-ajax-orch
+                              {:endpoint endpoints/get-inputs-outputs
+                               :payload (get-in response ["methodConfiguration" "methodRepoMethod"])
+                               :headers {"Content-Type" "application/json"}
+                               :on-done (fn [{:keys [success? get-parsed-response]}]
+                                            (if success?
+                                              (swap! state assoc :loaded-config response :inputs-outputs (get-parsed-response))
+                                              (swap! state assoc :error ((get-parsed-response) "message"))))}))
+                       (swap! state assoc :error status-text)))}))
+   :load-new-method-template
+   (fn [{:keys [state props refs this]} new-snapshot-id]
+     (let [namespace (get-in (:loaded-config @state) ["methodConfiguration" "methodRepoMethod" "methodNamespace"])
+           name (get-in (:loaded-config @state) ["methodConfiguration" "methodRepoMethod" "methodName"])
+           method-ref {"methodNamespace" namespace
+                       "methodName" name
+                       "methodVersion" new-snapshot-id}]
+       (swap! state assoc :blocker "Updating...")
+       (react/call :load-agora-method (@refs "methodDetailsViewer") {:namespace namespace
+                                                                     :name name
+                                                                     :snapshotId new-snapshot-id} state)
+       (endpoints/call-ajax-orch
+         {:endpoint (endpoints/create-template method-ref)
+          :payload method-ref
+          :headers {"Content-Type" "application/json"}
+          :on-done (fn [{:keys [success? get-parsed-response status-text]}]
+                       (if success?
+                         (let [response (get-parsed-response)]
+                              (endpoints/call-ajax-orch
+                                {:endpoint endpoints/get-inputs-outputs
+                                 :payload (get-in response ["methodRepoMethod"])
+                                 :headers {"Content-Type" "application/json"}
+                                 :on-done (fn [{:keys [success? get-parsed-response]}]
+                                            (swap! state dissoc :blocker)
+                                            (let [template {"methodConfiguration" (assoc response "namespace" namespace
+                                                                                         "name" name)}]
+                                              (if success?
+                                                (swap! state assoc :loaded-config (assoc template
```

Comment:
this constructs a method configuration using just a template. since templates don't yet have validated inputs/outputs, i'm shoving those fields onto the template so the UI can display properly. there are no inputs/outputs to validate at this point anyway so this will work as intended